plugins/github/server/uninstall.ts

```diff
@@ -9,7 +9,8 @@ export async function uninstall(
     const installationId = integration.settings?.github?.installation.id;
 
     if (installationId) {
-      return githubApp.deleteInstallation(installationId);
+      const client = await GitHub.authenticateAsInstallation(installationId);
+      await GitHub.requestAppUninstall(installationId, client);
```

Comment:
This wrapper has ended up kind of unnecessary, and it's a bit of an awkward DX to pass `client` as second param – could just use octokit directly here?

Reply:
Can use directly, yes. I just stuck to keeping all `octokit.request()` calls within the class with the intent of not having to import Octokit in other places. Just trying to set a convention but flexible, yes.

Reply:
Lets use this pattern and extend the client perhaps:

https://github.com/octokit/core.js/#defaults

Will allow doing `await client.requestAppUninstall()`